shared/redux/store.ts

```diff
@@ -0,0 +1,13 @@
+import { configureStore } from "@reduxjs/toolkit";
+import { api } from "./rtk-apis/tasksAPI";
```

Comment:
```suggestion
import { configureStore } from "@reduxjs/toolkit";

import { api } from "./rtk-apis/tasksAPI";
```

Should be a line gap between library and relative imports